diff --git a/node_modules/jitsi-meet-electron-utils/screensharing/main.js b/node_modules/jitsi-meet-electron-utils/screensharing/main.js
index 4f427fc..5c16d2e 100644
--- a/node_modules/jitsi-meet-electron-utils/screensharing/main.js
+++ b/node_modules/jitsi-meet-electron-utils/screensharing/main.js
@@ -116,18 +116,6 @@ class ScreenShareMainHook {
      * @param {string} bundleId- OSX Application BundleId
      */
     _verifyScreenCapturePermissions(bundleId) {
-        const {
-            hasPromptedForPermission,
-            hasScreenCapturePermission,
-            resetPermissions,
-        } = require('mac-screen-capture-permissions');
-
-        const hasPermission = hasScreenCapturePermission();
-        const promptedAlready = hasPromptedForPermission();
-
-        if (promptedAlready && !hasPermission) {
-            resetPermissions({ bundleId });
-        }
     }
 }
 
